src/frontend/App/context/EkspanderbareVilkårpanelContext.tsx

```diff
@@ -0,0 +1,59 @@
+import constate from 'constate';
+import { useState } from 'react';
+
+export enum EkspandertTilstand {
+    EKSPANDERT = 'EKSPANDERT',
+    KOLLAPSET = 'KOLLAPSET',
+}
+
+export enum EInngangsvilkår {
+    MEDLEMSKAP = 'MEDLEMSKAP',
+    OPPHOLD = 'OPPHOLD',
+    MOR_ELLER_FAR = 'MOR_ELLER_FAR',
+    NYTT_BARN_SAMME_PARTNER = 'NYTT_BARN_SAMME_PARTNER',
+    SIVILSTAND = 'SIVILSTAND',
+    SAMLIV = 'SAMLIV',
+    ALENEOMSORG = 'ALENEOMSORG',
+}
+
+const [EkspanderbareVilkårpanelProvider, useEkspanderbareVilkårpanelContext] = constate(() => {
+    const settAlleTil = (tilstand: EkspandertTilstand) =>
```

Comment:
```suggestion
    const settAlleTilTilstand = (tilstand: EkspandertTilstand) =>
```
Kan vurderes. Metoden blir mer beskrivende men navnet blir samtidig rart.

Reply:
Hmm ja var usikker på navnet selv, men når den brukes blir det f.eks. `settAlleTil(KOLLAPSET)` og da hjelper inputen på hvor beskrivende den er. Men kunne kanskje dratt inn noe om at det er panelene det gjelder evt. Samme gjelder isåfall ` lukkAlle()` og `åpneAlle()`